tests/test_helpers.go

```diff
@@ -0,0 +1,144 @@
+package tests
+
+import (
+	"context"
+	"crypto/ecdsa"
+	"math/big"
+	"runtime"
+	"strings"
+	"testing"
+
+	"github.com/ethereum/go-ethereum/accounts/abi/bind"
+	"github.com/ethereum/go-ethereum/core/types"
+	ethcrypto "github.com/ethereum/go-ethereum/crypto"
+	"github.com/ethereum/go-ethereum/ethclient"
+	"github.com/stretchr/testify/require"
+
+	"github.com/athanorlabs/go-relayer/common"
+)
+
+/*
+ * Golang packages can be tested in parallel, but in our case, they are sharing a
+ * common ganache Ethereum simulator. To avoid transaction conflicts, we allocate
+ * each package listed in `testPackageNames` one or more keys not shared with the
+ * tests of any other package.
+ */
+
+// testPackageNames is the list of packages with _test.go code that requires access to
+// one or more prefunded ganache wallets.
+var testPackages = []struct {
+	name    string
+	numKeys int
+}{
+	{"impls/gsnforwarder", 1},
+	{"impls/mforwarder", 1},
+}
+
+const (
+	repoName = "github.com/athanorlabs/go-relayer/"
+)
+
+// `ganache --deterministic` provides the following keys with 1000 ETH on startup.
+var ganacheTestKeys = []string{
+	"4f3edf983ac636a65a842ce7c78d9aa706d3b113bce9c46f30d7d21715b23b1d", // ganache key #0
+	"6cbed15c793ce57650b9877cf6fa156fbef513c4e6134f022a85b1ffdd59b2a1", // ganache key #1
+	"6370fd033278c143179d81c5526140625662b8daa446c22ee2d73db3707e620c", // ganache key #2
+	"646f1ce2fdad0e6deeeb5c7e8e5543bdde65e86029e2fd9fc169899c440a7913", // ganache key #3
+	"add53f9a7e588d003326d1cbf9e4a43c061aadd9bc938c843a79e7b4fd2ad743", // ganache key #4
+	"395df67f0c2d2d9fe1ad08d1bc8b6627011959b79c53d7dd6a3536a33ab8a4fd", // ganache key #5
+	"e485d098507f54e7733a205420dfddbe58db035fa577fc294ebd14db90767a52", // ganache key #6
+	"a453611d9419d0e56f499079478fd72c37b251a94bfde4d19872c44cf65386e3", // ganache key #7
+	"829e924fdf021ba3dbbc4225edfece9aca04b929d6e75613329ca6f1d31c0bb4", // ganache key #8
+	"b0057716d5917badaf911b193b12b910811c1497b5bada8d7711f758981c3773", // ganache key #9
+}
```

Comment:
I capped it at 10 keys (the default), so you only need to pass the `--accounts=N` flag.